specification/nhs-website-content-api.yaml

```diff
@@ -30,40 +30,24 @@ info:
       - Ratings and Reviews API
 
     ## API status and roadmap
-    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase. If you are an existing user, you need to sign up for the API in the new API Platform.
+    This API has moved from the nhs.uk developer API website. It is currently in development. If you are an existing user, you need to sign up for the API in the new API platform.
 
     ## Service level
 
-    This API is a gold service, meaning that it is operational and supported 24 x 7 x 365
-
-    For more details, see [service levels](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#service-levels).
+    This API is a [gold service](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#service-levels), meaning that it is operational and supported 24 x 7 x 365
 
     ## Technology
     This API is [RESTful](https://digital.nhs.uk/developer/guides-and-documentation/our-api-technologies#basic-rest).
 
-    page objects conform to global [schema.org](https://schema.org/) standards.
-
-    The content API is driven by the main nhs.uk Wagtail cms application. An endpoint exists for all pages underneath the homepage.
-
-    ## Endpoints
-    The main sections of the site are:
-    - /conditions
-    - /medicines
-    - /live-well
-    - /mental-health
-    - /common-health-questions
-    - /pregnancy
-    - /nhs-services
-
-    All pages under these urls also have endpoints
+    Page objects conform to global [schema.org](https://schema.org/) standards.
 
     ## Network access
     No special network access is required for this API. It is available over the internet.
 
     ## Security and authorisation
     This API is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
-    The end user will need to provide a API key in the header or as a query parameter in requests.
+    You must provide a API key in the header or as a query parameter in requests.
```

Comment:
"or as a query parameter in requests"
Will this still be possible?

Reply:
I think so but I can remove it and then ask in the design review